### ISSUE TITLE
Can we remember hit location modifiers

### ISSUE THREAD

**Issue by crnormand** [crnormand/gurps#1403]
To add to damage chat

If the player hits vitals, can we apply "vitals" to the damage chat?

**Comment by crnormand** [comment]
Thinking in the broader sense... maybe we can keep a TO DO list.   Some kind of state that indicates what would trigger it, and what action occurs when triggered.

In the simplest form, if someone rolls an attack, and they have added a location modifier (from the character sheet, or pulldown menu in the modifier tooltip), we could add "set location to: XXXX" as the action, and the trigger would be "when calculating damage".

Of course, this may be overkill... but I am trying to think ahead.   I know we have had other situations like this (I just can't remember them at the moment).   

It would work for "Deceptive attack" as well.   Assuming the player hits, they could assign a "-X to active defense" action when trying to roll an active defense.   Of course... this is a level of automation that could get tricky.    We could automatically apply it to an NPC, and the GM wouldn't have to do anything.   We might even be able to display it as a temporary status (as a visual reminder).   Here it is crossing over into the "effects" world... would it be better as an Active Effect?   Yeah, probably.

So does that mean the "to hit" location might be better as an active effect?   yeah, that might be true also.    

We just don't know enough yet.   So I am going to table this for now.   We might be able to dream up a lot of cool things, once we have the active effects code in place.

**Comment by mjeffw** [comment]
I don't think this requires active effects. I think it requires knowing that a modifier is a hit location modifier. If we can tell that, we can extract the hit location and send it to the damage chat.
